wp-content/themes/redmond/web/scss/app.scss

```diff
@@ -32,9 +32,14 @@ h1, h2, h3, h4, h5, h6 {
     margin-bottom: rem-calc(8px);
     padding-top: rem-calc(30px);
     text-transform: none;
+    white-space: nowrap;
```

Comment:
remove this line. if words are too long/big we will handle them a different way on smaller screens